packages/go/dawgs/drivers/neo4j/transaction.go

```diff
@@ -55,6 +56,14 @@ type neo4jTransaction struct {
 	traversalMemoryLimit size.Size
 }
 
+func (s *neo4jTransaction) WithGraph(graphSchema graph.Graph) graph.Transaction {
```

Comment:
unused method param; is this intentional or is there something we should be doing with the input graph schema?

Reply:
This needs a comment. Neo4j does not support multiple graphs.

Reply:
Note added